lib/highlight.js

```diff
@@ -21,13 +21,16 @@ function highlightUtil(str, options = {}) {
   hljs.configure({ classPrefix: useHljs ? 'hljs-' : ''});
 
   const data = highlight(str, options);
+  const lang = options.lang || data.language || '';
+  const classNames = (useHljs ? 'hljs' : 'highlight') + (lang ? ` ${lang}` : '');
 
   if (useHljs && !gutter) wrap = false;
+  if (gutter && !wrap) wrap = true; // arbitrate conflict ("gutter:true" takes priority over "wrap:false")
```

Comment:
I update this PR.  I added a code to arbitrate conflict with `gutter:` and `wrap:`.
I think this arbitration will be meaningful for adding `wrap:` option into Hexo's `_config.yml`.

@curbengh Could you review again?

Reply:
Somehow hexo binary is broken when installing from this PR, can you try rebasing?

Reply:
Can you add a test case for `"gutter: true, wrap false"`?